Client/Frontend/Brave Today/Cards/BraveTodayWelcomeView.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Lottie
+import BraveUI
+import Shared
+
+enum WelcomeCardAction {
```

Comment:
Kept it out because we reference it in `BraveTodaySectionProvider` instead of only in this view and its action handler. Could change it to `BraveTodayWelcomeView.Action` if preferred